docs/config.html

```diff
@@ -348,6 +357,88 @@ <h3 class="mt1 f6 lh-title" id="parse.gitfunctions">GitFunctions</h3>
   </ul>
 </section>
 
+<section class="mt4">
+  <h2 id="plugindefinition" class="title-2">[PluginDefinition]</h2>
+
+  <p>Used in repos that implement a Please plugin. By adding this section to your <code class="code">.plzconfig</code>,
+    your project can be included as a plugin in other Please repos.
+  </p>
+
+  <p>The following sets up a plugin with ID foo.</p>
+
+  <pre class="code-container">
+    <!-- prettier-ignore -->
+    <code>
+    [PluginDefinition]
+    Name = foo
+
+    [PluginConfig "FOO_TOOL"]
```

Comment:
I'm not sure about FOO_TOOL here. I feel like it should be `"foo_tool"` in this context and `CONFIG.FOO_TOOL` in asp?

Reply:
Ya, I think this can be made consistent with how the line above `Name = foo` works, where it's referenced via `CONFIG.FOO`